### ISSUE TITLE
Diagram is not rendering in live preview mode

### ISSUE THREAD

**Issue by Gourav1910** [zapthedingbat/drawio-obsidian#81]
Please check that what is going wrong with it because it is not rendering in live preview mode which is very important otherwise I can't see it while writing my notes.

**Comment by Andrea-28-03** [comment]
You just need to refresh it removing and putting it back or closing and reopening the note it fix it i don't know if there is a faster workaround, but this work for me

**Comment by shaunc** [comment]
Yes .. this is pretty non-intuitive. I wish there was a "refresh" menu option or (better) icon.
